data/basics/overflow.go

```diff
@@ -200,17 +200,10 @@ func (t *OverflowTracker) ScalarMulA(a MicroAlgos, b uint64) MicroAlgos {
 // Muldiv computes a*b/c.  The overflow flag indicates that
 // the result was 2^64 or greater.
 func Muldiv(a uint64, b uint64, c uint64) (res uint64, overflow bool) {
-	var aa big.Int
-	aa.SetUint64(a)
-
-	var bb big.Int
-	bb.SetUint64(b)
-
-	var cc big.Int
-	cc.SetUint64(c)
-
-	aa.Mul(&aa, &bb)
-	aa.Div(&aa, &cc)
-
-	return aa.Uint64(), !aa.IsUint64()
```

Comment:
```
Uint64 returns the uint64 representation of x. If x cannot be represented in a uint64, the result is undefined.
```
in theory it might break some not well written smart contract

Reply:
The function isn't used in the AVM, so we're safe on that front.